### PR TITLE
[BUGFIX] Fix the build with the latest PHP_CodeSniffer

### PR DESCRIPTION
**Pull request by Argee88**
I hope this resolves Issue #188 . If something's still missing, please report. If not I'm looking forward to 1.0.0

### CODE REVIEW

File: src/Csv/FormatInterface.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
```
Comment (oliverklee) [inline src/Csv/FormatInterface.php]: @Argee88 This looks like you added the two new files instead of adding the blank lines. What happened here?

Comment (oliverklee) [reply src/Csv/FormatInterface.php]: Possibly you'll need to rebase your branch from master first, resolve any conflicts and then force-repush.

Comment (Argee88) [reply src/Csv/FormatInterface.php]: Rebased and pushed again... I hope it works this time :/

**Comment by oliverklee** [comment]
@Argee88 No, the merge conflicts are still there … are you sure that you rebase and didn't merge master into your branch?

**Comment by Argee88** [comment]
yes, i'll try a second branch

**Comment by oliverklee** [comment]
Closed in favor of #190.

**Pull request closed without merging**